app/controllers/concerns/audit_log.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+module AuditLog
+  extend ActiveSupport::Concern
+  VALID_METHODS = [:info, :warn, :debug, :error]
+
+  private
+
+  def auditLog(user, action, args)
+    auditLogger(:info, user, action, args)
+  end
+
+  def auditInfo(user, action, args)
+    auditLogger(:info, user, action, args)
+  end
+
+  def auditWarn(user, action, args)
```

Comment:
method names should be snake case `audit_warn` ... also should not include a bunch of methods into everything ... prefer Audit.warn or something like that